src/components/Board.js

```diff
@@ -16,18 +18,118 @@ class Board extends Component {
     };
   }
 
+  componentDidMount = () => {
+    // console.log('Component did mount was called');
+    // axios.get('').then().catch();
+    axios.get(this.props.url + this.props.boardName + '/cards')
+    .then( (response) => {
+      console.log(response); //to see how the data looks like and decide what we want.
+      this.setState({
+        cards: response.data
+      });
+    })
+    .catch( (error) => {
+      this.setState({
+        error: error.message
+      })
+    });
+  }
+
+  renderCardList = () => {
+    // console.log(`this.state.cards = ${this.state.cards}`)
+    const componentList = this.state.cards.map((element, index) => {
+
+      console.log(element.card.text)
+      return (
+        <Card
+        key={index}
+        index={index}
+        id={element.card.id}
+        text={element.card.text}
+        emoji={element.card.emoji}
+        deleteCardCallback={this.deleteCard}
+        />
+      );
+    });
+    return componentList;
+  }
+
+  deleteCard = (id, index) => {
+    const url = this.props.url + this.props.boardName + '/cards/'+id
+    console.log(id);
+    axios.delete(url)
+    .then((response) => {
+      console.log(response);
+      const updatedCards = this.state.cards
+      delete updatedCards[index]
```

Comment:
Just a note, `delete` doesn't reindex the array so if you do

```javascript
delete myArray[0];
```

Then the first element of the array will be `undefined`.